common/lib/xmodule/xmodule/modulestore/tests/test_mongo.py

```diff
@@ -229,9 +223,6 @@ def test_get_courses(self, _from_json):
                 ['edX', 'test_unicode', '2012_Fall'],
                 ['edX', 'toy', '2012_Fall'],
                 ['guestx', 'foo', 'bar'],
-                # This course below is due to a caching issue in the modulestore
-                # which is not cleared between test runs.
-                ['TestX', 'ChildTest', '1234_A1'],
```

Comment:
This course is created in `common/lib/xmodule/xmodule/modulestore/tests/test_mongo.py` and is still actively used. The race condition present in this test, however, no longer is.